hummingbot/strategy/pure_market_making/pure_market_making.pyx

```diff
@@ -90,6 +89,7 @@ cdef class PureMarketMakingStrategy(StrategyBase):
                  status_report_interval: float = 900,
                  minimum_spread: Decimal = Decimal(0),
                  hb_app_notification: bool = False,
+                 order_override: {} = None,
```

Comment:
This should be something like Dict[xxx, xxx] where xxx is the type, e.g. str, not {}.